src/postgres/getPostgresTablesQueryRows.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ClientConfig } from "pg";
+import { runPostgresQuery, wrapArgInQuotes } from "./runPostgresQuery";
+
+export interface IPostgresTable {
+    schemaName: string;
+    name: string;
+    oid: number;
+    columnNames: string[];
+}
+
+function getTableInfo(): string {
+    return `select schemaname, tablename
```

Comment:
I feel like @wwlorey should take over reviewing this PR since he wrote most of the existing pg queries in this repo. The rest of the code looks fine to me, but not that familiar with the pg queries themselves

For example, I wasn't sure if there was a way to combine the queries into one, more-efficient query